cmake/projects/ittapi/hunter.cmake

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
```

Comment:
extend to 2021 or maybe add an additional line with just Rahul Sheth, @rbsheth what do you think?